source/sass/patterns/molecules/nav-bar.scss

```diff
@@ -0,0 +1,46 @@
+@import "../../mixins/media-query";
+@import "../../mixins/scale";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+@import "../../settings/font";
+
+$nav-bar-font-family: $font-secondary !default;
+$nav-bar-font-weight: bold !default;
+$nav-bar-item-spacing-default: 15px !default;
+$nav-bar-item-spacing-medium: 2 * $nav-bar-item-spacing-default !default;
+$nav-bar-letterspacing: 0.5px !default;
+$nav-bar-scale: -1 !default;
+$nav-bar-uppercase: true !default;
+
+.nav-bar {
+  @include block-spacing($end: 0);
+  font-family: $nav-bar-font-family;
+  font-weight: $nav-bar-font-weight;
+  @include set-font-size-and-line-height(scale($nav-bar-scale));
+  letter-spacing: $nav-bar-letterspacing;
+  @if $nav-bar-uppercase {
+    text-transform: uppercase;
+  }
+}
+
+.nav-bar__list {
```

Comment:
Are we missing the height in this as it shows height of 3.375rem in the eLife site or are we going to use margin-end or padding-end?